cmd/bridge/main.go

```diff
@@ -19,12 +19,12 @@ import (
 
 	"github.com/openshift/console/pkg/auth"
 	"github.com/openshift/console/pkg/bridge"
-	"github.com/openshift/console/pkg/crypto"
```

Comment:
Can you remove `pkg/crypto` from console if it's no longer needed?

Reply:
yeah, we probably should!

Reply:
The line you you are mentioning was changed but the second commit:
![Screenshot 2020-08-11 at 15 44 31](https://user-images.githubusercontent.com/1668218/89904791-a75d6480-dbe9-11ea-9613-c26b649e1bb0.png)